core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java

```diff
@@ -34,7 +34,6 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
-
```

Comment:
Why did you remove this line ? [spec](https://github.com/codenvy/codenvy-dev-resources/wiki/Source-file#imports)